packages/server/__tests__/arpa_reporter/server/services/persist-upload.spec.js

```diff
@@ -34,36 +62,54 @@ describe('persist-upload', () => {
     });
 
     describe('createUploadRow', () => {
-        const createUploadRow = persistUpload.__get__('createUploadRow');
+        const createUploadRow = persistUploadModule.__get__('createUploadRow');
 
         it('should create an upload row with escaped notes', async () => {
-            const body = { notes: '<script>alert("xss");</script>' };
-            const user = { id: 1 };
-            const reportingPeriod = { id: 2 };
-            const uploadRow = await createUploadRow('filename.xlsx', reportingPeriod, user, body);
-            expect(uploadRow.filename).to.equal('filename.xlsx');
-            expect(uploadRow.reporting_period_id).to.equal(2);
-            expect(uploadRow.user_id).to.equal(1);
-            expect(uploadRow.notes).to.equal('&lt;script&gt;alert(&quot;xss&quot;);&lt;/script&gt;');
+            const notes = '<script>alert("xss");</script>';
+            const userId = 1;
+            const reportingPeriodId = 2;
+            const agencyId = 2;
+            const uploadRow = await createUploadRow('filename.xlsx', reportingPeriodId, userId, agencyId, notes);
+            expect(uploadRow).to.deep.equal({
+                filename: 'filename.xlsx',
+                reporting_period_id: 2,
+                user_id: 1,
+                agency_id: 2,
+                notes: '<script>alert("xss");</script>',
+            });
         });
     });
 
     describe('persistUploadToFS', () => {
-        const persistUploadToFS = persistUpload.__get__('persistUploadToFS');
+        const persistUploadToFS = persistUploadModule.__get__('persistUploadToFS');
         let fsMock;
         const filename = 'test.xlsx';
         const upload = { filename, id: 1 };
         const filePath = uploadFSName(upload);
         const buffer = Buffer.from('test data');
 
+        beforeEach(() => {
```

Comment:
Is this duplicate beforeEach call (and the duplicate afterEach below) intentional? It looks like maybe a merge error somewhere resulted in two copies of both

Reply:
Oof, good catch, got this when resolving merge conflicts.